src/holidays/united-states/federal-reserve-bank/is-federal-reserve-bank-holiday.ts

```diff
@@ -0,0 +1,25 @@
+// holidays/united-states/federal-reserve-bank/is-federal-reserve-bank-holiday.ts
```

Comment:
not sure i like this file name. it's returning more than just whether it's a federal reserve bank holiday or not. it's also returning whether it's a weekend or not.

Reply:
a weekend is a US federal reserve bank holiday.